flink-runtime/src/test/java/org/apache/flink/runtime/webmonitor/TestingRestfulGateway.java

```diff
@@ -269,13 +270,13 @@ public Builder setRequestClusterOverviewSupplier(Supplier<CompletableFuture<Clus
 			return this;
 		}
 
-		public Builder setRequestMetricQueryServicePathsSupplier(Supplier<CompletableFuture<Collection<String>>> requestMetricQueryServicePathsSupplier) {
-			this.requestMetricQueryServicePathsSupplier = requestMetricQueryServicePathsSupplier;
+		public Builder setRequestMetricQueryServiceGatewaysSupplier(Supplier<CompletableFuture<Collection<MetricQueryServiceGateway>>> requestMetricQueryServiceGatewaysSupplier) {
+			this.requestMetricQueryServiceGatewaysSupplier = requestMetricQueryServiceGatewaysSupplier;
 			return this;
 		}
 
-		public Builder setRequestTaskManagerMetricQueryServicePathsSupplier(Supplier<CompletableFuture<Collection<Tuple2<ResourceID, String>>>> requestTaskManagerMetricQueryServicePathsSupplier) {
-			this.requestTaskManagerMetricQueryServicePathsSupplier = requestTaskManagerMetricQueryServicePathsSupplier;
+		public Builder setRequestTaskManagerMetricQueryServiceGatewaysSupplier(Supplier<CompletableFuture<Collection<Tuple2<ResourceID, MetricQueryServiceGateway>>>> requestTaskManagerMetricQueryServiceGatewaysSupplier) {
```

Comment:
typo: remove s in `Gateways`, also applies to field and  ´requestTaskManagerMetricQueryServiceGateways`

Reply:
Given the return value is `Collection<Tuple2<ResourceID, MetricQueryServiceGateway>>` and means return the gateways of TaskManagers' MQS, I'd like to keep "s". However, we have outdate document of `requestTaskManagerMetricQueryServiceGateways` that should be updated.